lib/sie_exporter.rb

```diff
@@ -0,0 +1,117 @@
+# encoding: utf-8
+
+require 'sie'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.